GeometryCore/src/Triangle.cpp

```diff
@@ -0,0 +1,35 @@
+#include "../include/LinearAlgebra.h"
+#include "../include/Triangle.h"
+
+namespace geom_utils
+{
+
+	FPoint3D getNormal(const Triangle3D& triangel)
+	{
+		//compute vector
+		FPoint3D normal = geom_utils::cross(triangel.a, triangel.b);
```

Comment:
Don't think this is absolutely correct.
Firstly, you should calsulate vectors: vecA = b - a; vecB = c - a; Then vecA x vecB.

Reply:
done